library_generation/test/integration_tests.py

```diff
@@ -49,6 +51,28 @@
 
 
 class IntegrationTest(unittest.TestCase):
+    def test_get_commit_message_success(self):
+        repo_url = "https://github.com/googleapis/googleapis.git"
+        baseline_commit = "a17d4caf184b050d50cacf2b0d579ce72c31ce74"
+        description = generate_pr_descriptions(
+            f"{config_dir}/google-cloud-java/{config_name}",
```

Comment:
Can we add another test for split repos? Like `java-bigtable`?

Reply:
Done.